inst/workflow/1_mappings/STUDY.yaml

```diff
@@ -45,18 +45,54 @@ steps:
     params:
       data: Temp_CTMSStudyWide
       strGroupLevel: "Study"
+  - output: Temp_CTMSplanned
+    name: RunQuery
+    params:
+      df: Raw_STUDY
+      strQuery: "SELECT studyid as GroupID, num_plan_site as SiteTarget, num_plan_subj as ParticipantTarget FROM df"
   - output: Temp_StudyCountsWide
     name: RunQuery
     params:
       df: Mapped_SUBJ
       strQuery: "SELECT studyid as GroupID, COUNT(DISTINCT subjid) as ParticipantCount, COUNT(DISTINCT invid) as SiteCount FROM df GROUP BY studyid"
-  - output: Temp_StudyCounts
+  - output: Temp_CountTargetsWide
+    name: left_join
+    params:
+      x: Temp_CTMSplanned
+      "y": Temp_StudyCountsWide
+      by: GroupID
+  - output: Temp_CountTargetsPercsWide
+    name: RunQuery
```

Comment:
I wasn't involved in the conversations here, but would it be easier to just write a utll function that converts these to percentages instead of using sql? this works, so its fine if this is the approach we prefer here, but just thinking about our use of `MakeLongMeta()` etc for using dplyr to do some of these conversions

Reply:
I was thinking it would be easier to use dplyr as well but I didn't know how to accomodate for the `...` nature into the output/name/params structure presented here and writing an extra function felt like adding "code bloat" that didn't feel necessary considering it is "SQL-able"